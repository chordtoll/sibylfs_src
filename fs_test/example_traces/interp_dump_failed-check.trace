# processing file 'example_traces/interp_warn_dump_failed-int.trace' ...
        @type trace
     1: dump /
