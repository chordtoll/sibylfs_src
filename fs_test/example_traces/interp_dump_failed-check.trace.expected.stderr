
Fatal error: File "example_traces/interp_dump_failed-check.trace", line 3, characters 5-15:
dump not allowed in trace
